### PR TITLE
Release 0.0.4

### PR DESCRIPTION
**Pull request by SylvainCorlay**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ requirements:
     - pip
   run:
     - python
-    - voila
+    - voila>=0.1,<0.2
```
Comment (scopatz) [inline recipe/meta.yaml]: This needs a space, `voila >=0.1,<0.2`

**Comment by scopatz** [comment]
Thanks @SylvainCorlay! I think the sha256 sum also needs to be updated.

**Comment by scopatz** [comment]
here is the updated sha256: `e9c61e4859732c2ffcf6ce217889c05e28fd4862d469e7d4d1e11d565dbf2530`

**Comment by SylvainCorlay** [comment]
> Thanks @SylvainCorlay! I think the sha256 sum also needs to be updated.

Thanks I forgot to push :)

File: recipe/meta.yaml (hunk 1 above)
Comment (SylvainCorlay) [reply recipe/meta.yaml]: fixed.

**Comment by SylvainCorlay** [comment]
Ready to go.

**Pull request merged**